priceloop_api/priceloop/extras/table.py

```diff
@@ -0,0 +1,855 @@
+import json
+from enum import Enum
+from http import HTTPStatus
+from io import StringIO
+from typing import Any, Callable, List, Literal, Optional
+
+import attr
+import numpy as np
+import pandas as pd
+import requests
+from pandas.api.types import infer_dtype
+from retry import retry
```

Comment:
In one tech discussion we talked about replacing retry, because it is not really maintained anymore (last update about 8 years ago) and quite limited